src/lib/constants.ts

```diff
@@ -97,8 +97,8 @@ export const SEVEN_DAYS_IN_SECONDS = 7 * 24 * 60 * 60;
 
 export const ETHERSCAN_LIST_NAME = 'EtherscanList';
 
-export const ETHERSCAN_PATH = process.env.PWD + '/src/FullList/all_tokens.json';
-export const ALL_TOKENS_GOERLI_ROLLUP_PATH =
-  process.env.PWD + '/src/FullList/421613_all_tokens.json';
-export const TOKENLIST_DIR_PATH = process.env.PWD + '/src/ArbTokenLists';
-export const FULLLIST_DIR_PATH = process.env.PWD + '/src/FullList';
+export const OUTPUT_FOLDER = process.env.PWD + '/build';
```

Comment:
If this were put on the current live digital ocean droplet, it would break it, right? 

If so / for the transition, let's just have dedicated branches to minimize confusion. E.g.,
- On digital ocean, branch off of current `master` branch to create a branch called `digital-ocean-version`
- merge this onto `master`; `master` becomes "s3 version"
- when we're ready, move the endpoints from DO to s3

Reply:
(or is the plan to switch directly to s3 at the same time that this is merged?)

Reply:
@chrstph-dvx I can imagine we can do this in a way that works on both environments?